engines/stark/ui/menu/mainmenu.cpp

```diff
@@ -0,0 +1,104 @@
+#include "engines/stark/ui/menu/mainmenu.h"
+
+namespace Stark {
+
+MainMenuScreen::MainMenuScreen(Gfx::Driver *gfx, Cursor *cursor) :
+        StaticLocationScreen(gfx, cursor, "MainMenuLocation", Screen::kScreenMainMenu) {
+}
+
+MainMenuScreen::~MainMenuScreen() {
+}
+
+void MainMenuScreen::open() {
+    StaticLocationScreen::open();
+
+    //TODO: Implement each handler
+    _widgets.push_back(new StaticLocationWidget(
+			"BGImage",
+			nullptr,
+			nullptr));
+    
+    _widgets.push_back(new StaticLocationWidget(
+			"NewGame",
+			nullptr,
+			nullptr));
+	_widgets.back()->setupSounds(0, 1);
```

Comment:
Please fix the whitespaces here. We use tabs for indentation and spaces for alignment (see http://wiki.scummvm.org/index.php/Code_Formatting_Conventions).